.gitignore

```diff
@@ -0,0 +1,6 @@
+notify/
```

Comment:
Note you have some `.pyc` files in this PR. Please remove the `.pyc` files and update `.gitignore` so it won't happen again.

Reply:
@GeetikaBatra please add also `*.pyc` to .gitignore

Reply:
Done